examples/todocontroller/deployment.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: todo-controller-staging
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: todo-controller-staging
+    spec:
+      containers:
+      - name: todo-controller
+        image: us.gcr.io/not-my-project/todo-controller:staging
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 50051
+        env:
+          - name: MY_NAMESPACE
```

Comment:
You can also get this from /var/run/secrets/kubernetes.io/serviceaccount/namespace and avoid using the downward API.

Reply:
neat, but I'd rather avoid reading a file :-/